newIDE/app/src/MainFrame/EditorContainers/HomePage/TeamSection/TeamGroupNameField.js

```diff
@@ -0,0 +1,174 @@
+// @flow
```

Comment:
you have 2 TeamGroupNameField & NewTeamGroupNameField components. I'm wondering if they could be merged into one with a specific prop to handle the difference?

Reply:
Currently, I prefer having two separate components. I fear merging them would end up having a hard to read component